TestModels/Aggregate/runtimes/go/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/Smithy-dafny/TestModels/Aggregate
```

Comment:
All of these changes are already in the branch and reviewed. Are you sure you are not force pushing?

Reply:
Not at all. If I was force pushing, it would mention 
```
rishav-karanjit force-pushed the ...
```
Like in this PR: https://github.com/aws/aws-database-encryption-sdk-dynamodb/pull/1266

Reply:
I wonder its a bug inside github because I merged `rishav-checkin-go-mod` into 2 branches. I needed to merge it in rishav-union when you were in vacation to get unblocked.
- rishav-union in PR #4
- scchatur/GoCodegen in PR #6 

But I will look into it.